connman/include/provider.h

```diff
@@ -48,6 +48,7 @@ enum connman_provider_state {
 	CONNMAN_PROVIDER_STATE_READY         = 3,
 	CONNMAN_PROVIDER_STATE_DISCONNECT    = 4,
 	CONNMAN_PROVIDER_STATE_FAILURE       = 5,
+	CONNMAN_PROVIDER_STATE_WAIT_USER     = 6,
```

Comment:
This state is really `association` and perhaps it would be a good name for it anyways. Was thinking also `WAIT_AGENT`.

Reply:
Changed.